fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -109,6 +112,14 @@ dropViewStatement
     : DROP VIEW (IF EXISTS)? qualifiedName
     ;
 
+// ------------------------------------------- Task Statement ----------------------------------------------------------
+
+submitTaskStatement
+    : SUBMIT TASK qualifiedName?
+    sessionPropertis?
```

Comment:
Do we need to support session in this version?

Reply:
I think user can change session variable by /* SET_VAR= */. we don't need to give them more options

Reply:
I agree with you